dvc/api/experiments.py

```diff
@@ -31,3 +32,43 @@ def make_checkpoint():
         os.fsync(fobj.fileno())
     while os.path.exists(signal_file):
         sleep(0.1)
+
+
+def make_experiment(
```

Comment:
It should probably be named `save_experiment` or `create_experiment`.

Reply:
Since we went with `params_show` to mimic the cli, what about `exp_save`?

Reply:
I don't like `params_show()` naming tbh (what are we _showing_?).

Reply:
In cli, we use `dvc <noun> <verb>` and function names are usually a verb, hence it makes sense for me to name it as `save_experiment` rather than `exp_save`.

Reply:
@skshetry We need to keep consistency. We had this argument in `params_show` before.

Reply:
It's a different world tbh, there's no reason to force CLI in API, and make APIs a second-class citizen.

And, why are they inconsistent?  `save_experiments() == "exp save"`, no?

Reply:
Let's not block this and since we have `params_show`, let's go with `exp_save`.

I said the same arguments as @skshetry when adding `params_show`.
We can make an alias in the future, it is not that important.